text/0000-localizing-api-messages.md

```diff
@@ -0,0 +1,140 @@
+
+* Start Date: 2022-03-29
+* RFC PR: 
+* FOLIO Issue: 
+* Submitter : Radhakrishnan Gopalakrishnan (rgopalakrishnan@ebsco.com)
+* Co-Submitter(s) : Zak Burke (zburke@ebsco.com)
+* Sub Group : 
+  * Marc Johnson (marc.johnson@k-int.com)
+  * Julian Ladisch (julian.ladisch@gbv.de)
+  * Peter Murray (peter@indexdata.com)
+  * Tod Olson (tod@uchicago.edu)
```

Comment:
I'm not sure we should classify folks who provided feedback (and some who attended a meeting to discuss that feedback) as a working group.